lib/lit.rb

```diff
@@ -37,18 +40,16 @@ def self.init
 
   def self.check_if_table_exists
     Lit::Locale.table_exists?
-  rescue
```

Comment:
Are you sure it's not needed? I think it was needed when there was no database created yet or something like that.